.github/CODEOWNERS

```diff
@@ -77,3 +77,40 @@ Makefile                    @getsentry/owners-sentry-dev
 /tests/sentry/snuba/test_tasks.py                               @getsentry/workflow
 /tests/snuba/snuba/test_query_subscription_consumer.py          @getsentry/workflow
 ### Endof Issue/Metric Alerts ###
+
+
+### Perfomance/Discover ###
+/src/sentry/snuba/                                              @getsentry/visibility
+/tests/sentry/snuba/test_discover.py                            @getsentry/visibility
+
+/src/sentry/discover/                                           @getsentry/visibility
+
+/src/sentry/utils/snuba.py                                      @getsentry/visibility
+
+/src/sentry/api/event_search.py                                 @getsentry/visibility
```

Comment:
Hmm `api/event_search.py` might be bigger than just visibility and [the last matching pattern takes the most precedence.](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners)

Reply:
Well we can fight with whoever else wants it I guess. If it's an issue we can make a new team called "search" or something and use that.